src/main/java/backend/IssueMetadata.java

```diff
@@ -33,6 +33,15 @@ public IssueMetadata(IssueMetadata other) {
         this.isUpdated = other.isUpdated;
     }
 
+    // Copy constructor used in reconciliation
+    public IssueMetadata(IssueMetadata other, boolean isUpdated) {
+        this.events = new ArrayList<>(other.events);
+        this.comments = new ArrayList<>(other.comments);
+        this.nonSelfUpdatedAt = other.nonSelfUpdatedAt;
+        this.nonSelfCommentCount  = other.nonSelfCommentCount;
+        this.isUpdated = isUpdated;
```

Comment:
Just thought it was a bit vague. I guess it's fine, since I don't have any ideas for a more precise or obvious name... So in other words, it tells us whether or not the metadata is valid, and if not, that it should be replaced by valid metadata?